components/brave_wallet/browser/BUILD.gn

```diff
@@ -639,6 +641,26 @@ generated_types("meld_integration_responses") {
   ]
 }
 
+source_set("blockchain_images_srouce_base_unit_tests") {
+  testonly = true
+  sources = [ "blockchain_images_source_base_unittest.cc" ]
+  deps = [
+    ":browser",
+    ":test_support",
+    "//base",
+    "//base/test:test_support",
+    "//testing/gtest",
+  ]
+}
+
+if (is_ios) {
+  bundle_data("brave_wallet_test_data_bundle") {
+    testonly = true
+    sources = [ "//brave/test/data/brave_wallet/1.0.1/images/logo.png" ]
```

Comment:
this file should be moved to `data = [ ... ]` in `blockchain_images_srouce_base_unit_tests`